runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -8899,12 +8901,141 @@ mod centrifuge {
 			assert_ok!(
 				pallet_transfer_allowlist::Pallet::<T>::add_transfer_allowance(
 					RawOrigin::Signed(account.into()).into(),
-					asset,
+					FilterCurrency::Specific(asset),
 					location
 				)
 			);
 		}
 
+		fn restrict_all<T: Runtime>() {
+			let mut env = RuntimeEnv::<T>::from_parachain_storage(
+				Genesis::default()
+					.add(genesis::balances::<T>(cfg(TRANSFER_AMOUNT + 10)))
+					.add(orml_tokens::GenesisConfig::<T> {
+						balances: vec![(
+							Keyring::Alice.to_account_id(),
+							USDC,
+							T::ExistentialDeposit::get() + usdc(TRANSFER_AMOUNT),
+						)],
```

Comment:
I think there is already a one line utility for just this in `genesis` module

Reply:
Yeah, but that does not work well with the new `add(..)` logic here.